src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs

```diff
@@ -8,7 +8,12 @@
              :border-gray "#cacaca"
              :text-light "#7f7f7f"
              :text-gray "#666"
-             :footer-text "#989898"})
+             :footer-text "#989898"
+             :success-green "#7aac20"
+             :running-blue "#67688a"
+             :exception-red "#e85c46"
+             :tag-background "#d4ecff"
+             :tag-foreground "#2c3c4d"})
```

Comment:
This list is growing long-enough that it might make sense to alphabetize for easier scanning.